opencensus/proto/exporter/exporter.proto

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// NOTE: This proto is experimental and is subject to change at this point.
+// Please do not use it at the moment.
+
+package opencensus.proto.exporter;
+
+import "opencensus/proto/trace/trace.proto";
+import "opencensus/proto/stats/metrics/metrics.proto";
+
+option java_multiple_files = true;
+option java_package = "io.opencensus.proto.exporter";
```

Comment:
Suggestion: rename the package to "export" instead of "exporter", since in Java "exporter" points to the actual exporters (Stackdriver, Prometheus).

Reply:
What about proto exporter or something more specific?

Reply:
Let's leave it as it is, for consistency across language :)